package.json

```
@@ -9,7 +9,7 @@
     "clean": "rm -rf dist/",
     "dev": "nodemon",
     "dev:run": "yarn build && NODE_ENV=development node -r dotenv/config dist/index.js",
-    "docker:install": "node dist/lib/install.js",
+    "docker:install-inside": "node dist/lib/install.js",
```

It can actually be triggered outside of docker so I suggest (to update in Dockerfile too):
```suggestion
    "install-chromium": "node dist/lib/install.js",
```

---

tru